service/src/main/java/org/cbioportal/service/impl/TreatmentServiceImpl.java

```diff
@@ -170,4 +169,12 @@ private PatientTreatmentRow createPatientTreatmentRowForTreatment(
     public Boolean containsTreatmentData(List<String> samples, List<String> studies) {
         return treatmentRepository.getTreatmentCount(samples, studies) > 0;
     }
+
+    @Override
+    public Boolean containsSampleTreatmentData(List<String> samples, List<String> studies) {
```

Comment:
since `samples` is always null, why not remove it and pass null while invoking `getSampleCount` and `getTreatmentCount` methods?